src/Compute/custom/NewAzVmss_SimpleParameterSet.cs

```diff
@@ -0,0 +1,373 @@
+        public string AllocationMethod { get; set; } = "Static";
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string VnetAddressPrefix { get; set; } = "192.168.0.0/16";
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string SubnetAddressPrefix { get; set; } = "192.168.1.0/24";
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string FrontendPoolName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string BackendPoolName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false, HelpMessage = "Use this to add system assigned identity (MSI) to the vm")]
+        public SwitchParameter SystemAssignedIdentity { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false, HelpMessage = "Use this to add the assign user specified identity (MSI) to the VM")]
+        [ValidateNotNullOrEmpty]
+        public string UserAssignedIdentity { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public SwitchParameter EnableUltraSSD { get; set; }
+
+        [Parameter(
+            ParameterSetName = SimpleParameterSet,
+            Mandatory = false,
+            HelpMessage = "A list of availability zones denoting the IP allocated for the resource needs to come from.",
+            ValueFromPipelineByPropertyName = true)]
+        public List<string> Zone { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public int[] NatBackendPort { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public int[] DataDiskSizeInGb { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false, HelpMessage = "Use this to create the Scale set in a single placement group, default is multiple groups")]
+        public SwitchParameter SinglePlacementGroup;
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string ProximityPlacementGroup { get; set; }
+
+        const int FirstPortRangeStart = 50000;
+
+        sealed class Parameters : IParameters<VirtualMachineScaleSet>
+        {
+            NewAzVmss_SimpleParameterSet _cmdlet { get; }
+
+            Client _client { get; }
+
+            public Parameters(NewAzVmss_SimpleParameterSet cmdlet, Client client)
+            {
+                _cmdlet = cmdlet;
+                _client = client;
+            }
+
+            public string Location
+            {
+                get { return _cmdlet.Location; }
+                set { _cmdlet.Location = value; }
+            }
+
+            public ImageAndOsType ImageAndOsType { get; set; }
+
+            public string DefaultLocation => "eastus";
```

Comment:
@markcowl do we not want to follow the same default location logic as the VM simple parameter set?